app/models/order.rb

```diff
@@ -5,6 +5,10 @@ class Order < ActiveRecord::Base
   has_many :items, through: :order_details
   has_many :shipments
 
+  scope :by_status_includes_extras, lambda { |statuses, extras = [:organization, :order_details, :shipments]|
```

Comment:
Could you make this `def self.by_status_includes_extras(statuses, extras = [:organization, :order_details, :shipments])` instead? Class methods is the newer form of scopes, and grepping our codebase, I don't see any `scope` style scopes in use, so I think it would be nice to be consistent with that.

Reply:
Also, what do you think about automatically converting statuses to the enum value so the caller can just use symbols, like `Order.by_status_includes_extras(:closed)` or `Order.by_status_includes_extras([:rejected, :closed, :canceled])`.

If you want to not allow the number form at all (which would be simpler and clearer I think), you could implement it like: `statuses = [statuses].flatten.map { |s| Order.statuses[s] }`

This would make inlining the variables where this method is being used worthwhile I think.

Reply:
👍